manifest.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "Anti-Paywall",
+  "author": "nextgens",
   "version": "1.0",
   "description": "This extension will mangle your browser's requests to maximize the chances of bypassing paywalls.",
   "permissions": ["webRequest", "webRequestBlocking", "*://*/*"],
   "background": {
-    "scripts": ["background.js"]
+    "scripts": ["background.js"],
+    "persistent": false
```

Comment:
`background.persistent` is required for Edge and Chrome [cites `false`](https://developer.chrome.com/extensions/background_pages#manifest) for background scripts (and I assume this is default).

Firefox outputs this error message upon side-loading, but otherwise the extension works fine. It comes across as an error, but is actually a WARN. [See Bugzilla issue on changing this message](https://bugzilla.mozilla.org/show_bug.cgi?id=1370077). I don't think this affects the user install experience, but I'm unfamiliar with that process.

![image](https://user-images.githubusercontent.com/297483/46317171-3cc85000-c5a0-11e8-9918-904f33cda132.png)